Composer/packages/tools/language-servers/language-generation/src/utils.ts

```diff
@@ -76,13 +76,13 @@ export function convertDiagnostics(lgDiags: LGDiagnostic[] = [], document: TextD
 
 export function textFromTemplate(template: Template): string {
   let text = '';
```

Comment:
Use `[]` to build a string is faster than concatenating strings. But I am okay to keep it today.

```js
const { parameters = [] } = template;
let textBuilder = [];

textBuilder.push(`# ${template.name.trim()}`);

if (parameters.length) {
  textBuilder.push(`(${parameters.join(', ')})`;
}

textBuilder.push(`\n${template.body.trim()}`);

return textBuilder.join('');
```